src/plugin/plugins.ts

```diff
@@ -26,4 +27,5 @@ export var plugins: markdowItPlugin[] = [
     { plugin: require('markdown-it-mark') },
     { plugin: require('markdown-it-deflist') },
     { plugin: MarkdownItExportHelper }
+    { plugin: MarkdownItMultiMdTable }
```

Comment:
```suggestion
    { plugin: require('markdown-it-multimd-table'), params: [{ enableMultilineRows: true }] },
```
You don't have to create a plaugin file `markdownItMultiMdTable.ts`.

Reply:
You are right the single line solution is much nicer than the extra plugin file